app/controllers/sessions_controller.rb

```diff
@@ -39,6 +39,6 @@ def find_user
   end
 
   def new_spire
-    (User.pluck(:spire).map(&:to_i).last + 1).to_s.rjust 8, '0'
+    ((User.pluck(:spire).map(&:to_i).last + 1).to_s.rjust 8, '0') + '@umass.edu'
```

Comment:
The parentheses that were originally omitted were around the `rjust`, not the entire expression.